.github/workflows/ci.yml

```diff
@@ -152,10 +152,10 @@ jobs:
     - name: Set LIBCLANG_PATH
       if: startsWith(matrix.os, 'windows')
       run: echo "LIBCLANG_PATH=$((gcm clang).source -replace "clang.exe")" >> $env:GITHUB_ENV
-    - if: startsWith(matrix.os, 'windows')
+    - if: "startsWith(matrix.os, 'windows') && !contains(matrix.target, 'ios')"
       # CI's Windows doesn't have require root certs
       run: cargo test --workspace --exclude tokio-boring --exclude hyper-boring
       name: Run tests (Windows)
-    - if: "!startsWith(matrix.os, 'windows')"
+    - if: "!startsWith(matrix.os, 'windows') && !contains(matrix.target, 'ios')"
```

Comment:
Why this change? If it's because the tests were failing, why didn't they fail before this PR?

Reply:
It's not possible to run `cargo test` when building for iOS. It would produce test binaries, but they can't be executed on the host macOS. You'd need to start an iOS Simulator, wrap the binaries into iOS apps, upload the apps to the simulator, run them there, extract results, and post those back. That's too much hassle. I believe it's okay to skip this step until Rust ecosystem catches up.

So for iOS builds the tests are skipped, only checking that `boring-sys` is able to compile BoringSSL and that rustc can compile Rust code that should be using the library.

Reply:
Also, this reminds me. When CI does `cargo test`, it compiles for the host. That is, I believe that tests for Android targets are actually testing on the host Linux – not the Android 🤔